### DISCUSSION TITLE
Mass duplicate bot comments?

### DISCUSSION THREAD

**Discussion by Wesbalt** [community/community#31968] [Other Feature Feedback, Questions, & Ideas]
Hi, I am working on a project on pull requests, specifically about the events that happen to them, and I discovered something strange I think. On rare occasion, bots spam the pull request thread with duplicate comments.

Example A: https://github.com/openshift/origin-aggregated-logging/pull/2053

Example B: https://github.com/openshift/origin-aggregated-logging/pull/2211

Example C: https://github.com/ethereum/remix-project/pull/1182

I'm not a seasoned GitHub user so I don't know what to make of this. Could someone please tell me if this is expected behavior or some kind of bug? Thank you.

**Comment by airtower-luna** [comment]
The expectation is really something to ask operators and users of the bot about. Though looking at the examples I'd assume that a bot that writes that many comments without it being desired would quickly be removed. :wink:
